lib/crawler_web/controllers/keyword_controller.ex

```diff
@@ -23,6 +23,11 @@ defmodule CrawlerWeb.KeywordController do
     end
   end
 
+  def show(conn, %{"id" => keyword_id}) do
+    keyword = Keywords.get_keyword_by_id(keyword_id)
```

Comment:
should we fetch the user's keyword 🤔 with just the `keyword_id` without that any user can see other user's keyword

Don't forget the test for it

```suggestion
    keyword = Keywords.get_keyword_by_user_id_and_id(current_user.id, keyword_id)
```

Reply:
Fixed [9b25756](https://github.com/vnntsu/elixir-ic/pull/43/commits/9b25756b2ca0de85d0e517dc53f4ead7494039ad) and added system test too.